inference-engine/src/mkldnn_plugin/memory_desc/cpu_blocked_memory_desc.cpp

```diff
@@ -108,10 +122,13 @@ size_t CpuBlockedMemoryDesc::getMaxMemSize() const {
         return getCurrentMemSize();
     }
 
-    auto& maxDims = shape.getMaxDims();
-    if (std::any_of(maxDims.begin(), maxDims.end(), [](size_t x){ return Shape::UNDEFINED_DIM == x ||
+    auto maxDims = shape.getMaxDims();
+    if (shape.hasZeroDims()) {
+        // in getCurrentMemSize we check that desc is defined otherwise return UNDEFINED SIZE
+        std::replace(maxDims.begin(), maxDims.end(), static_cast<Dim>(Shape::UNDEFINED_DIM), static_cast<Dim>(0));
```

Comment:
We can't do this because of `CpuBlockedMemoryDesc::getCurrentMemSizeImp` implementation. See: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/mkldnn_plugin/memory_desc/cpu_blocked_memory_desc.cpp#L96

Reply:
No we can't. Because of implementation `CpuBlockedMemoryDesc::getCurrentMemSizeImp()` compute size differently and I guess better to call `getCurrentMemSize()` in all situation instead return 0.
Also I noticed that we have difference between `getCurrentMemSize()` which return `UNDEF` in case if at least one dim `UNDEF`, but also at least one dim equal 0. And `getMaxMemSize()` which return 0 in this case.
So I aligned this behavior and now  `getCurrentMemSize()` also return 0